packages/adapter-xata/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "@next-auth/xata-adapter",
+  "version": "0.1.0",
+  "description": "Xata adapter for next-auth.",
+  "homepage": "https://next-auth.js.org",
+  "repository": "https://github.com/nextauthjs/next-auth",
+  "bugs": {
+    "url": "https://github.com/nextauthjs/next-auth/issues"
+  },
+  "author": "Tejas Kumar",
+  "main": "dist/index.js",
+  "license": "ISC",
+  "keywords": [
+    "next-auth",
+    "next.js",
+    "oauth",
+    "xata"
+  ],
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc",
+    "preversion": "build",
+    "prepublish": "build"
```

```suggestion
    "prepublish": "build",
    "test": "echo 'Test skipped'"
```

Let's add a dummy test for now.